packages/platform-core/API.md

```diff
@@ -166,12 +166,12 @@ export const packageJsonSchema: z.ZodObject<{
     type: z.ZodOptional<z.ZodUnion<[z.ZodLiteral<"module">, z.ZodLiteral<"commonjs">]>>;
 }, "strip", z.ZodTypeAny, {
     name?: string | undefined;
-    type?: "module" | "commonjs" | undefined;
     version?: string | undefined;
+    type?: "module" | "commonjs" | undefined;

```

Comment:
same in this file.

Reply:
Updated in [091f133](https://github.com/aws-amplify/amplify-backend/pull/2269/commits/091f133b18a0bc04843ef8ed6b39d46efe550ce1)